src/resolvers/mutation/create-deployment/index.js

```diff
@@ -70,6 +70,22 @@ export default async function createDeployment(parent, args, ctx, info) {
     throw new WorkspaceSuspendedError();
   }
 
+  // Filter out needed keys
+  const keys = [
+    "scheduler",
+    "webserver",
+    "statsd",
+    "pgbouncer",
+    "flower",
+    "redis",
+    "workers",
+    "executor"
+  ];
+
+  const deploymentArgs = pick(args, keys);
+  const deploymentConfigData = await createDeploymentConfig(deploymentArgs);
```

Comment:
```suggestion
  const deploymentConfigData = createDeploymentConfig(deploymentArgs);
```

We don't need await here do we?

Reply:
Yeah, we dont. Forgot to clean that up.